static/scripts/files-storage.js

```diff
@@ -157,66 +139,109 @@ $(document).ready(() => {
 			});
 	}
 
-	function updateUploadProcessingProgress() {
-		if (!progressBarActive) {
-			$progress.css('width', '0%');
+	function fileNameEditClickHandler(e) {
+		e.stopPropagation();
+		e.preventDefault();
+		const fileRecordId = $(this).attr('data-file-id');
+		const oldName = $(this).attr('data-file-name');
+		$renameModal.find('#newNameInput').val(oldName);
+		$renameModal.find('#fileRecordId').val(fileRecordId);
+		$renameModal.modal('show');
+	}
 
-			$form.fadeOut(50, () => {
-				$progressBar.fadeIn(50);
-			});
+	// eslint-disable-next-line array-callback-return
+	$filesStorageComponents.map((_index, element) => {
+		const $filesStorageComponent = $(element);
+		const $form = $filesStorageComponent.find('.form-files-storage');
+		const getDataValue = (attr) => {
+			const value = $filesStorageComponent.find('.section-upload').data(attr);
+			return value || undefined;
+		};
+
+		const getSchoolId = getDataValue('school');
```

Comment:
```suggestion
		const schoolId = getDataValue('school');
```
Gilt auch für viele andere get-Funktionen in diesem Controller.